en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -1,163 +1,174 @@
 # Configuring a JDBC User Store
 
 User management functionality is provided by default in all WSO2
-Carbon-based products and is configured in the user-mgt.xml file found
-in the \<PRODUCT\_HOME\>/repository/conf/ directory. This file is
-shipped with user store manager configurations for all possible user
-store types (JDBC, read-only LDAP/Active Directory, read-write LDAP and
-read-write Active directory). The instructions given below explains how
-to configure an RDBMS (JDBC) as the primary user store for the WSO2
-server.
+Carbon-based products and is configured in the
+`         deployment.toml       ` file found in the
+`         <IS_HOME>/repository/conf/        ` directory. This
+documentation explains the process of setting up a primary user store
```

Comment:
... documentation explains the process of setting up **the** primary user store ...?